libs/ec/inc/public/tfc/cia/402.hpp

```diff
@@ -0,0 +1,127 @@
+#pragma once
+
+#include <boost/sml.hpp>
+
+namespace tfc::ec::cia_402 {
+
+enum commands_e : uint16_t {
```

Comment:
we should use strict typing, either `enum struct` or `enum class` so it cannot be implicitly converted to any type that can be implicitly converted to uint16_t